apax/model/builder.py

```diff
@@ -15,21 +19,44 @@ def __init__(self, model_config: ModelConfig, n_species: int = 119):
         self.n_species = n_species
 
     def build_basis_function(self):
-        basis_fn = GaussianBasis(
-            n_basis=self.config["n_basis"],
-            r_min=self.config["r_min"],
-            r_max=self.config["r_max"],
-            dtype=self.config["descriptor_dtype"],
-        )
+
+        basis_config = self.config["basis"]
+        name = basis_config["name"]
+
+        if name == "gaussian":
+            basis_fn = GaussianBasis(
+                n_basis=basis_config["n_basis"],
+                r_min=basis_config["r_min"],
+                r_max=basis_config["r_max"],
+                dtype=self.config["descriptor_dtype"],
+            )
+        elif name == "bessel":
+            basis_fn = BesselBasis(
+                n_basis=basis_config["n_basis"],
+                r_max=basis_config["r_max"],
+                dtype=self.config["descriptor_dtype"],
+            )
+        else:
+            raise NotImplementedError("unknown basis requested")
         return basis_fn
 
     def build_radial_function(self):
         basis_fn = self.build_basis_function()
+
+        if self.config["basis"]["name"] == "gaussian":
+            use_embed_norm = True
+            one_sided_dist = False
```

Comment:
Is there a scenario where both can be `true`or `false` - maybe in the future? If not, although the names are more descriptive, I'd vote for a single variable name to not allow for errors.

Reply:
In principle yes